make/test.mk

```
@@ -30,10 +30,13 @@ endif
 
 E2E_TEST_EXECUTION ?= true
 
+LETS_ENCRYPT_PARAM_VALUE := false
+
 ifeq ($(IS_OSD),true)
 LETS_ENCRYPT_PARAM := --lets-encrypt
```

I think you should set it to true here right? I mean you're not setting it to true anywhere, unless I'm missing something.
```suggestion
LETS_ENCRYPT_PARAM := --lets-encrypt
LETS_ENCRYPT_PARAM_VALUE=true
```

---

or rather this:
```
ifneq ($(IS_OSD),true)
LETS_ENCRYPT_PARAM := --lets-encrypt=false
```

---

it should be that way.. i was just trying to make it work in the CI.. 
changed it now.